lib/Entity/Widget.php

```diff
@@ -1269,6 +1269,9 @@ private function linkMedia()
                 'mediaId' => $mediaId,
                 'mediaId2' => $mediaId
             ));
+
+            // audit the media being added
+            $this->getLog()->audit('Media', $mediaId, 'Media Added', ['mediaId' => $mediaId, 'widgetId' => $this->widgetId]);
```

Comment:
This will get us a message whenever a media item is linked to a widget, which is a good start. I think the message could make that clearer - i.e. "Media added to Widget".